packages/cli/test/functional/test_site_convert/test_basic_convert/expected/markbind/glyphicons/fonts/glyphicons-halflings-regular.svg

```diff
@@ -1,288 +1,288 @@
-<?xml version="1.0" standalone="no"?>
```

Comment:
Sorry, I've just marked svg files as text, I think we missed this out in https://github.com/MarkBind/markbind/pull/1683.

Try merging in the latest changes and see if you still get these line ending differences